api/services/tagPresentation.js

```diff
@@ -0,0 +1,31 @@
+export const fetchAndPresentTagJoins = (joinClass, communityId, userId) =>
```

Comment:
the reason I didn't call it TagPresenter is because it's a collection of functions, not a class.
You want me to put the functions in a class called TagPresenter, or just rename the file TagPresenter?